README.md

```diff
@@ -1,2 +1,56 @@
-# community
-Longhorn Community
+# Longhorn Community
+
+Focuses on supporting the Longhorn Cloud native distributed block storage built on and for Kubernetes
+
+## Meetings
+* Regular Meeting: [Every 2nd Friday of the Month at 12:00 Eastern Standard Time (EST)](https://zoom.us/j/710325604) (monthly). [Convert to your timezone](http://www.thetimezoneconverter.com/?t=12:00&tz=Eastern%20Standard%20Time%20%28EST%29).
+  * [Meeting notes and Agenda](https://docs.google.com/document/d/1QD119jBX1h544ZQ9E4XaititEMfq8qEByx-42MtRC-g/).
+  * [Meeting recordings](https://www.youtube.com/channel/UCGk1f-LCVmccf1pX4OvF1Ig).
+
+## Leadership
+
+### Chairs
+The Chairs of the group run operations.
+
+* Jason van Brackel (**[@jasonvanbrackel](https://github.com/JasonvanBrackel)**), Rancher Labs
+* TBD
+
+### Technical Leads
+The Technical Leads establish new subprojects, decommission existing
+subprojects, and resolve cross-subproject technical issues and decisions.
+
+* Sheng Yang (**[@yasker](https://github.com/yasker)**), Rancher Labs
+* TBD
+
+## Contact
+- Slack: [#longhorn-storage](https://rancher-users.slack.com/messages/longhorn-storage)
+- [Mailing list](https://groups.google.com/forum/#!forum/longhorn-io)
+- [Open Community Issues/PRs](https://github.com/longhorn/longhorn/issues)
+
+## Subprojects
+
+The following subprojects are owned by the longhorn organization:
+### longhorn
+
+### longhorn-ui
+
+### longhorn-engine
+
+### longhorn-instance-manager
+
+## Getting Started
+
+If you're looking for information getting started with longhorn see [Longhorn](https://github.com/longhorn/longhorn)
+
+## Discuss
+Join us on [Slack](https://rancher-users.slack.com/messages/longhorn-storage) or the [Rancher Forums](https://forums.rancher.com/c/longhorn) to chat about Longhorn.
+
+## Contributing
+
+To get yourself familiar with contributing, please read the Please read [Code of Conduct](https://github.com/longhorn/longhorn/blob/master/CODE_OF_CONDUCT.md) and [Contributing Guidelines](https://github.com/longhorn/longhorn/blob/master/CONTRIBUTING.md) before contributing.
+
+As always, you can also find the active list of open issues and proposals [Open Code Issues and PRs for Longhorn](https://github.com/longhorn/longhorn/issues) on GitHub.
+
+## Meeting Recordings
```

Comment:
Already mentioned above?

Reply:
Leaving this as is as it's common practice in K8s sigs